src/test/resources/geofencing_zones.json

```diff
@@ -380,6 +380,11 @@
                   ]
                 ]
               ]
+            ],
+            "encodedCoordinates": [
+              [
+                "{_keAwq`nJ}SzGi`AlWudAt_@ciAhzAtR~QrEbHnVpc@fBJl@|@pOv@b@`@|eJ~b@xuDppCvQrs@kaAdVjy@|{AzaE`mAlcGfOj\\cPluAxMb`Aza@npCoq@fhBkEpiA}bBvrBqqAifAiv@ytBsd@dl@ybBq~Ak|AtBazBe~A__A~~B_GzMwa@FsMx@uD`QmAZmFyDmDkBcHiXoVqDa[hCuo@naAkdAtwBqPiRmm@c~Bid@gBiLvXwFtHy[pG_Otf@q`@`bAaHd_@ww@xlAbe@l`Dw~EmiIacByo@~o@ivAk^gcBLooFlfAmkErcB}fBdcBw\\oMb`B_rCaeBmvAd{CguCcY}yB_mD__CcwLfeCixOxfFnNrkCtxAbs@hBhgEvMrKuCbL~MDjBpBjHxBnJtH_EhEpfB`gBiYhf@ci@eT{_B|n@qHng@vtE`rAvsChjA"
```

Comment:
You're probably not testing what you think you're testing. This test fixture is a GBFS file and should conform to the GBFS standard. You should add a test to check that the query for encoded coordinates returns the expected output.

Reply:
Yea. Tested here instead: https://github.com/entur/lamassu/pull/403/files#diff-0f86989bdc97351078de404478230ea75834c73fdf9a8a2a8c00ab38688bfbce